lib/core/capabilities/DoNotDisturbCapability.js

```diff
@@ -0,0 +1,39 @@
+const Capability = require("./Capability");
+const NotImplementedError = require("../NotImplementedError");
+
+class DoNotDisturbCapability extends Capability {
+    /**
+     * This function polls the current consumables state and stores the attributes in our robotState
+     *
+     * @abstract
+     * @returns {Promise<Array<import("../../entities/state/attributes/DoNotDisturbAttribute")>>}
+     */
+    async getDnd() {
+        throw new NotImplementedError();
+    }
+
+    /**
+     * @abstract
+     * @returns {Promise<void>}
+     */
+    async deleteDnd() {
```

Comment:
Remove this

Reply:
deleteDND retained but renamed for consistency to deleteDNDConfiguration.  The only way to disable DND on Roborocks is to send close_dnd_timer as set_dnd_timer only accepts start hour / minute and end hour / minute.

Reply:
Hm but what about looking at the `enabled` property of the `ValetudoDNDConfiguration` and selecting the correct miio command accordingly?